### PR TITLE
docs: create python version of ST recipe of abandoned carts (fixes #9913)

### LINKED ISSUE #9913: Create a Python version of our ST Recipe for Abandoned Carts
## Description ✍️ 

Take this [existing handler](https://hasura.io/docs/latest/scheduled-triggers/recipes/abandoned-cart/) written in JavaScript and provide a Python version.

## Acceptance criteria 📋 

- [ ] Use the `<Tabs />` component to create parity with a Python version of the handler already on this page ☝️

### PR DESCRIPTION
**Pull request by SoumyoPurkayastha**
### Description

Added a python version for the JS handler in docs for the ST recipe of abandoned carts. Also added the missing bash commands for the JS handler in the docs

### Changelog

__Component__ : docs 

__Type__: enhancement 

__Product__: community-edition

#### Short Changelog

Added python handler for Abandoned Carts ST recipe

### Related Issues

Close #9913 

#### Catalog upgrade

Does this PR change Hasura Catalog version?
- [x] No
- [ ] Yes
  - [ ] Updated docs with SQL for downgrading the catalog 

#### Metadata

Does this PR add a new Metadata feature?
- [x] No
- [ ] Yes
  - Does `run_sql` auto manages the new metadata through schema diffing?
    - [ ] Yes
    - [ ] Not required
  - Does `run_sql` auto manages the definitions of metadata on renaming?
    - [ ] Yes
    - [ ] Not required
  - Does `export_metadata`/`replace_metadata` supports the new metadata added?
    - [ ] Yes
    - [ ] Not required

#### GraphQL
- [x] No new GraphQL schema is generated
- [ ] New GraphQL schema is being generated:
   - [ ] New types and typenames are correlated
   
   

#### Breaking changes

- [x] No Breaking changes
- [ ] There are breaking changes:

  1. Metadata API

     Existing `query` types:
     - [ ] Modify `args` payload which is not backward compatible
     - [ ] Behavioural change of the API
     - [ ] Change in response `JSON` schema
     - [ ] Change in error code
     

  2. GraphQL API

     Schema Generation:
     
     
     - [ ] Change in any `NamedType`
     - [ ] Change in table field names
     

     Schema Resolve:-
     
     - [ ] Change in treatment of `null` value for any input fields 
     

  3. Logging

     - [ ] Log `JSON` schema has changed
     - [ ] Log `type` names have changed

### CODE REVIEW

**Comment by SoumyoPurkayastha** [comment]
Hey @robertjdominguez could you please review this PR and suggest changes if needed

File: docs/docs/scheduled-triggers/recipes/abandoned-cart.mdx [hunk 1]
```diff
@@ -287,6 +304,137 @@ nodemailer.createTestAccount((err, account) => {
...
+        recipient = user_email
+
+        msg = MIMEMultipart()
+        msg['From'] = sender
+        msg['To'] = recipient
+        msg['Subject'] = subject
+        msg.attach(MIMEText(body, 'plain'))
+
+        # send the email
+        server.sendmail(sender, recipient, msg.as_string())
```
Comment (robertjdominguez) [inline docs/docs/scheduled-triggers/recipes/abandoned-cart.mdx:358]:
```suggestion
     with smtplib.SMTP(smtp_server, smtp_port) as server:
          try:
              server.starttls()
              server.login(smtp_user, smtp_pass)
          except Exception as e:
              print(f"An error occurred while creating the SMTP server: {e}\nWe'll print the message to the terminal\n")
    
          # compose the email
          subject = f"You've left items in your cart, {user_name.split(' ')[0]}!"
          body = f"Hi {user_name.split(' ')[0]}, it seems you've left some items in your cart. Don't forget to check out!"
          sender = 'sender@SuperStore.com'
          recipient = user_email
    
          msg = MIMEMultipart()
          msg['From'] = sender
          msg['To'] = recipient
          msg['Subject'] = subject
          msg.attach(MIMEText(body, 'plain'))
    
          # send the email
          try:
              server.sendmail(sender, recipient, msg.as_string())
          except Exception as e:
              print(f"From: {msg['From']}\nTo: {msg['To']}\nSubject: {msg['Subject']}\n{body}")
```

Would you try a change like this to see if we can more elegantly handle users who don't have an SMTP server they can connect to? This helps illustrate the ease of use and [follows the convention here](https://github.com/Azanul/graphql-engine/blob/66c40f4ca31a42e74483ffa91f91616706618321/docs/docs/scheduled-triggers/recipes/daily-summary-email.mdx#L313-L320).

**Review by robertjdominguez — CHANGES_REQUESTED** [review]
Thanks for this @SoumyoPurkayastha — please check the feedback, try it out, and see if you agree 👍

**Comment by robertjdominguez** [comment]
Thanks, @SoumyoPurkayastha — I've approved this on our internal repo 🎉 

CI should pick this up shortly and merge it for you. We appreciate this contribution 🚀 

cc @Stefmore02

**Comment by SoumyoPurkayastha** [comment]
Thanks! Looking forward to making further contributions here 😁

**Comment by Stefmore02** [comment]
Hey @SoumyoPurkayastha!

Thanks so much for your contribution to Hasura OSS during Hacktoberfest 2023! 🎉 We'd love to send you some Hasura swag as a token of appreciation, please send me an email to estefani@hasura.io to send you our swag form.✨

**Pull request closed without merging**